### ISSUE TITLE
Battery not appearing with Kernel 3.19rc5-MANJARO

### ISSUE THREAD

**Issue by nuclearsandwich** [nuclearsandwich/surface3-archlinux#16]
Battery doesn't show up in KDE power management or `/sys/class/power`. I'll see if it returns with rc6 or reverting to 3.18.

The presence or absense of acpid doesn't seem to matter.

**Comment by nuclearsandwich** [comment]
Does work with 3.18.

**Comment by paulhandy** [comment]
can confirm with 3.19.0-1

**Comment by colorprint** [comment]
This patch works to resolve this issue:
https://dl.dropboxusercontent.com/u/4080500/patch/battery.patch

**Comment by GaaH** [comment]
Does somenone tried with the 3.19 ? It seems to be released in stable version

**Comment by colorprint** [comment]
Yes, the patch works with 3.19 (but bug still not fixed in the kernel).

Here is the kernel bug opened:
https://bugzilla.kernel.org/show_bug.cgi?id=90161
